source/fundamentals/versioned-api.txt

```diff
@@ -0,0 +1,123 @@
+=============
+Versioned API
+=============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   The Versioned API feature requires MongoDB Server 5.0 or later. You
+   should only use the Versioned API feature if you are connecting to
+   MongoDB servers that are upgraded to a version that supports this feature.
+
+This guide shows you how to specify the **Versioned API** when connecting to
+a MongoDB instance or replica set. You can use the Versioned API feature to
+force the server to run operations with consistent behavior as specified by
+an **API version**. An API version defines expected behavior of operations
+such as the information expected in the request and the response.
+
+When you use the Versioned API feature, you can update your MongoDB driver or
+server without worrying about backward compatibility issues of the commands
+covered by the Versioned API.
+
+..
+   TODO: See the server manual page on `Versioned API <https://docs.mongodb.com/v5.0/reference/versioned-api/>`__
+   for more information including the set the commands covered.
+
+The following sections describe how you can enable Versioned API for
+your MongoDB client and each of the options that you can specify.
+
+Enable Versioned API on a MongoDB Client
+----------------------------------------
+
+To enable the Versioned API, you must specify an API version in the settings
+that you use to create your MongoDB client. Once you instantiate a
+``MongoClient`` instance with a specified Versioned API, all commands you
+run with that client use that version of the Versioned API. If you need to
+run commands not covered by the Versioned API or with a different version of
+it, instantiate a separate client.
+
+The example below shows how you can instantiate a ``MongoClient`` that
+specifies version "1" of the Versioned API and connects to a server by
+performing the following operations:
+
+- Construct a ``ServerApi`` instance using the ``ServerApi.Builder``
+  helper class.
+- Construct a ``MongoClientSettings`` instance using the
+  ``MongoClientSettings.Builder`` class.
+- Specify a server to connect to using a ``ServerAddress`` instance.
+- Instantiate a ``MongoClient`` using the ``MongoClients.create()`` method
+  and pass your ``MongoClientSettings`` instance as a parameter.
+
+.. literalinclude:: /includes/fundamentals/code-snippets/VersionedApiExample.java
+   :start-after: start serverAPIVersion
+   :end-before: end serverAPIVersion
+   :language: java
+   :dedent:
+
+.. warning::
+
+   Make sure that when you specify an API version that you connect to a
+   version of MongoDB server that supports Versioned API. Otherwise, your
+   application may raise an exception when attempting to perform any operation
+   that sends a command to MongoDB. For example, a query to a server could
+   return an exception message that resembles the following:
```

Comment:
I had to read this section a few times to understand this warning. I think it can be simplified to something along the lines of:
```suggestion
   If you specify an API version to a MongoDB server that does not support 
   Versioned API, your application may raise an exception when attempting 
   to send a command to MongoDB. For example, a query to a server could
   return an exception message that resembles the following:
```

Reply:
Thanks, I agree it's confusing and needs to be reworded. Will fix to something similar to your suggestion.